bitnami/redmine/templates/cronjob.yaml

```
@@ -84,6 +84,12 @@ spec:
                 {{- end }}
               resources: {{- toYaml .Values.mailReceiver.resources | nindent 16 }}
               env:
+                - name: REDMINE_DATABASE_TYPE
+                {{- if eq .Values.databaseType "mariadb" }}
+                  value: "mariadb"
+                {{- else }}
+                  value: "postgresql"
+                {{- end }}
```

You can use the same statement as in deployment.yaml
```suggestion
             - name: REDMINE_DATABASE_TYPE
                value: {{ .Values.databaseType | quote }}
```

---

Hello @Mauraza,

Indeed, it's nicer that way :-)
Modifications done.